kotlin/internal/repositories/setup.bzl

```diff
@@ -23,32 +23,33 @@ def kt_configure():
         name = "kotlin_rules_maven",
         fetch_sources = True,
         artifacts = [
-            "org.jetbrains.kotlinx:atomicfu-js:0.13.1",
-            "org.jetbrains.kotlinx:kotlinx-coroutines-core-js:1.3.2",
-            "org.jetbrains.kotlinx:kotlinx-coroutines-core:1.1.1",
-            "org.jetbrains.kotlinx:kotlinx-serialization-runtime:0.14.0",
+            "org.jetbrains.kotlinx:atomicfu-js:0.14.0",
+            "org.jetbrains.kotlinx:kotlinx-coroutines-core-js:1.3.9",
+            "org.jetbrains.kotlinx:kotlinx-coroutines-core:1.3.9",
+            "org.jetbrains.kotlinx:kotlinx-serialization-runtime:0.20.0",
             "com.google.code.findbugs:jsr305:3.0.2",
-            "junit:junit:4.13-beta-3",
-            "com.google.protobuf:protobuf-java:3.6.0",
-            "com.google.protobuf:protobuf-java-util:3.6.0",
-            "com.google.guava:guava:27.1-jre",
-            "com.google.truth:truth:0.45",
-            "com.google.auto.service:auto-service:1.0-rc5",
-            "com.google.auto.service:auto-service-annotations:1.0-rc5",
+            "junit:junit:4.13",
+            "com.google.protobuf:protobuf-java:3.13.0",
```

Comment:
Is there a reason why these are rolled up into 1.4?

Reply:
Nope. I just update everything I found. Can revert and leave only kotlin and coroutines.

Reply:
Please -- best to keep things focused.

Reply:
Done.